modules/temp-deck/temp-deck-arduino/temp-deck-arduino.ino

```diff
@@ -55,21 +59,21 @@ float _offset_temp_diff = 0.0;
 
 // the intensities of the fan (0.0-1.0)
 #define FAN_HIGH 1.0
-#define FAN_LOW 0.4
+#define FAN_LOW 0.3
 #define FAN_OFF 0.0
 
-// model version 3.0 has differents fans, and requires on/off cycles (not PWM)
+// model versions 3.0+ & 4.0+ have a different fan that requires on/off cycles (not PWM)
 #define MAX_FAN_OFF_TIME 2000
-#define FAN_V3_0_LOW 0.95
+#define FAN_V3_V4_LOW 0.95
 unsigned long fan_on_time = 0;
 unsigned long fan_off_time = MAX_FAN_OFF_TIME;
 unsigned long fan_timestamp = 0;
 bool is_fan_on = false;
-bool is_v3_0_fan = false;
+bool is_v3_v4_fan = false;
 
-// model version 3.1 has same fans as v3.0, but pcb uses mosfet to PWM fan power
-#define FAN_V4_0_LOW 0.1
-bool is_v4_0_fan = false;
+// LED pins for model versions 3 (post 2018.10.15) & 4: red = 6, blue = 5
+// LED pins for model versions < 3 & 3.0 (pre- 2018.10.15) : red = 5, blue = 6
+bool is_blue_pin_5 = false;
```

Comment:
It looks like `is_blue_pin_5` is only used in `setup`, it can be a local variable there